activation/validation.go

```diff
@@ -35,11 +35,11 @@ func (v *Validator) Validate(minerID signing.PublicKey, nipost *types.NIPost, ex
 		return fmt.Errorf("invalid `Challenge`; expected: %x, given: %x", expectedChallenge, nipost.Challenge)
 	}
 
-	if nipost.PostMetadata.BitsPerLabel < v.cfg.BitsPerLabel {
+	if uint(nipost.PostMetadata.BitsPerLabel) < v.cfg.BitsPerLabel {
 		return fmt.Errorf("invalid `BitsPerLabel`; expected: >=%d, given: %d", v.cfg.BitsPerLabel, nipost.PostMetadata.BitsPerLabel)
 	}
 
-	if nipost.PostMetadata.LabelsPerUnit < v.cfg.LabelsPerUnit {
+	if uint(nipost.PostMetadata.LabelsPerUnit) < v.cfg.LabelsPerUnit {
```

Comment:
labels per unit here is uint64? you should cast the other variable then

Reply:
done. Also opened PR to have the same field types in post - https://github.com/spacemeshos/post/pull/61